tests/integration/test_dbapi_integration.py

```diff
@@ -775,6 +775,18 @@ def test_cancel_query(trino_connection):
     assert "Cancel query failed; no running query" in str(cancel_error.value)
 
 
+def test_close_cursor(trino_connection):
+    cur = trino_connection.cursor()
+    cur.execute("SELECT * FROM tpch.sf1.customer")
+    cur.fetchone()  # TODO (https://github.com/trinodb/trino/issues/2683) test with and without .fetchone
```

Comment:
Is this TODO still relevant ?

Reply:
That particular issue is closed. I have tested the same code when omitting the `cur.fetchone()` the query is still cancelled, however it is not showing up in the Trino UI. I will remove the comment in a preparatory commit.